main.go

```diff
@@ -315,6 +309,18 @@ func main() {
 				os.Exit(1)
 			}
 		}
+
+		if !disableAdmissionWebhook {
```

Comment:
do you want to merge this with line 298?

Reply:
Yep, my bad... Thanks! :+1: